app/views/application_drafts/edit.haml

```diff
@@ -1,3 +1,8 @@
+Note: The data type of 'explanation' supports markdown.
+- if @site_text.present?
+  For a quick tutorial, please click
+  = link_to 'here.', site_text_url(@site_text.name)
```

Comment:
I'd also have been fine with just writing

```ruby
site_text_url('markdown explanation')
```

and leaving out the instance variable, but what you've done is fine.